alacritty_terminal/src/term/mod.rs

```diff
@@ -762,6 +766,8 @@ impl<T> Term<T> {
     }
 
     pub fn new<C>(config: &Config<C>, size: SizeInfo, event_proxy: T) -> Term<T> {
+        debug_assert!(size.cols() >= Column(MIN_COLS));
+        debug_assert!(size.lines() >= Line(MIN_LINES));
```

Comment:
Why are these assertions necessary?

Reply:
I think we should make it clear (for both the computer and code readers) that here we have conditions that the arguments must meet.